lxd/storage/drivers/driver_cephfs.go

```diff
@@ -242,6 +242,7 @@ func (d *cephfs) Delete(op *operations.Operation) error {
 func (d *cephfs) Validate(config map[string]string) error {
 	rules := map[string]func(value string) error{
 		"cephfs.cluster_name":    validate.IsAny,
+		"cephfs.fscache":         validate.IsBool,
```

Comment:
This needs a validate.Optional() I think

Reply:
Indeed